src/checkpointing/checkpoint.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from datetime import datetime
+
+class Checkpoint:
+    def __init__(self):
+        self._file = "checkpoint.txt"
+
+    # This function will write to the checkpoint file
+    # if there is an existing checkpoint it will overwrite that checkpoint
+    # date_to_checkpoint : Will be the date of the last row processed, this should be a string from a row entry
+    # end_date: Will be the ending date for the range of entries being queried, this should be a string from a row entry
+    def write_to_file(self, date_to_checkpoint, end_date):
+        f = open(self._file, "w")  # Will create the file if it does not exists, if it does it overwrites
+        f.write(date_to_checkpoint + "," + end_date)
+        f.close()
+        return None
```

Comment:
Unless I'm mistaken, functions return None by default, so this line isn't necessary.